up42/utils.py

```diff
@@ -5,14 +5,13 @@
 import tempfile
 import tarfile
 
-import folium.plugins
-import geojson
-import geopandas as gpd
+import folium
+from folium.plugins import Draw
```

Comment:
Is this in requirements? or is it installed by default?

Reply:
default